examples/language_model/squeezebert/models/squeezebert-uncased/tokenizer_config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Don't need this file.

Reply:
![db5a14a6979a52d7b6fa06de8a07ae4](https://user-images.githubusercontent.com/43545086/131795091-b42c4a92-80bd-440b-a3be-b3ec9621890d.jpg)
I was told to upload all the relevant files. And he will delete all files after testing.

Reply:
OK, then wait until testing passes.